kore/src/Kore/Repl/Data.hs

```diff
@@ -119,6 +122,20 @@ newtype RuleName = RuleName
     { unRuleName :: String
     } deriving (Eq, Show)
 
+newtype ReplOutput =
+    ReplOutput
+    { unReplOutput :: [ReplOut]
+    } deriving (Eq, Show, Semigroup, Monoid)
+
+newtype PrintAuxOutput = PrintAuxOutput
```

Comment:
Could we add comments here explaining how `PrintAuxOutput` and `PrintKoreOutput` are generally used? It wasn't clear to me, and I had to scroll most of the way through the pull request to discover that `KoreOut` is piped through `kast`.